### ISSUE TITLE
inspector continuing to watch keyboard after hidden

### ISSUE THREAD

**Issue by blairmacintyre** [aframevr/a-painter#198]
We're using aframe to teach a high school class, and one of the students noticed that when they went into the inspector, did some things, and exited, if they moved around with the keyboard, objects would start getting duplicated.

It turned out that every time they hit "d"  (to move back) the inspector was duplicating the last thing that was selected when they exited the inspector.

This was in the lesson 3 project in A-Frame school, working on glitch.com after remixing, but I think it happens in any project.

**Comment by dmarcos** [comment]
Is this an inspector issue instead of a-painter?

**Comment by fernandojsg** [comment]
It's an inspector yep, I'm going to try to fix it today

**Comment by blairmacintyre** [comment]
@dmarcos you're right, I should have posted this there (and should have also checked to see if this happened in other aframe programs).  But, thanks both of you.
